org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/JDTLanguageServer.java

```diff
@@ -725,4 +725,7 @@ public JavaClientConnection getClientConnection() {
 		return client;
 	}
 
+	public void notify(String method, Object parameter) {
```

Comment:
I have tried to use this notification from VS Code, but lsp4j sends only the method property.
I think you need to create an object containing the method and parameter properties. See ActionableNotification - https://github.com/eclipse/eclipse.jdt.ls/blob/master/org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaClientConnection.java#L47.

Reply:
To make the test working I had to modify https://github.com/eclipse/eclipse.jdt.ls/blob/master/org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/AbstractProjectsManagerBasedTest.java#L93 by adding a case for the 2nd parameter. Can it be something similar to this or sending the only first arg is like a 'hardcoded' limitation?

Reply:
Try to use the notify method in vscode-java. See https://github.com/redhat-developer/vscode-java/blob/master/src/extension.ts#L108